packages/parser-css/src/parser.ts

```diff
@@ -1,3 +1,5 @@
+const safe = require('postcss-safe-parser');
```

Comment:
Any reason to use require instead of import?

Reply:
The usual: `postcss-safe-parser` returns a function.

Reply:
ok!